app/js/model/filter.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'jquery',
+  'backbone'
+], function ($, Backbone) {
+  return Backbone.Model.extend({
+    defaults: {
+      none: true,
+      done: false,
+      notDone: false
+    },
+
+    setFilterNone: function () {
+      this.set({none: true});
+
+      this.set({notDone: false});
+      this.set({done: false});
+    },
+
+    setFilterDone: function () {
+      this.set({done: true});
+
+      this.set({notDone: false});
+      this.set({none: false});
+    },
+
+    setFilterNotDone: function () {
+      this.set({notDone: true});
+
+      this.set({done: false});
+      this.set({none: false});
```

Comment:
As you see, it's kind of difficult to have 3 booleans and managing them. Can we do better ?